src/components/footer/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { graphql, useStaticQuery } from 'gatsby';
+import { StrapiFrontPage } from '../../../graphql-types';
+import FooterSection from './footerSection';
+import LogoIcon from '../../images/logo.inline.svg';
+
+const footerQuery = graphql`
+  {
+    allStrapiFrontPage {
+      nodes {
+        locale
+        footer_sections {
+          id
+          link_groups {
+            id
+            links {
+              id
+              url
+              phone_number
+              email
+              description
+              title
+            }
+            some_links {
+              id
+              facebook_url
+              instagram_url
+              twitter_url
+              youtube_url
+            }
+          }
+          title
+        }
+      }
+    }
+  }
+`;
+
+const currentLocale = 'fi';
+
+function Footer() {
+  const { allStrapiFrontPage } = useStaticQuery<{ allStrapiFrontPage: { nodes: StrapiFrontPage[] } }>(footerQuery);
+
+  const footerSections = allStrapiFrontPage.nodes.find((x) => x.locale === currentLocale)?.footer_sections;
```

Comment:
Can this cause errors if fetch for some-reason fails and we try to access object "nodes" of null/undefined?

Reply:
If the query fails catastrophically (query is wrong etc.) then the whole build fails, which is a good thing.
Then if there re no FrontPage nodes or one is found and it doesn't have footer_sections, then the footer renders as emtpty.